src/lib.rs

```diff
@@ -64,6 +62,7 @@ impl StarReference {
             settings,
         };
 
+        #[allow(clippy::arc_with_non_send_sync)]
```

Comment:
@adam-azarchs @macklin-10x Is ignoring this warning the correct thing to do?

Reply:
Probably not?

Reply:
It should either
1. Be Send/Sync safe
2. Be wrapped in a muted
3. Use an Rc rather than an Arc, if it's not safe to use from multiple threads, to avoid the overhead involved in atomic operations.

Reply:
Opened this PR to use a `Rc` rather than an `Arc`
- https://github.com/10XGenomics/orbit/pull/78

Reply:
After Chris's review, opened this PR:
- https://github.com/10XGenomics/orbit/pull/80